### PR TITLE
Update libusb and copyright to latest revision

### PR DESCRIPTION
**Pull request by Vidrep**

### CODE REVIEW

File: dev-libs/libusb/libusb-1.0.21.recipe [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	2009-2013 Pete Batard
+	2009-2013 Ludovic Rousseau
+	2010-2012 Michael Plante
+	2011-2013 Hans de Goede
+	2012-2013 Martin Pieuchot
+	2012-2013 Toby Gray"
+         2013-2015 Chris Dickens"
+LICENSE="GNU LGPL v2.1"
+REVISION="2"
+SOURCE_URI="https://github.com/libusb/libusb/archive/v1.0.21.tar.gz"
```
Comment (fbrosson) [inline dev-libs/libusb/libusb-1.0.21.recipe]: You might wish to use:
```
REVISION="1"
SOURCE_URI="https://github.com/libusb/libusb/archive/v$portVersion.tar.gz"
```

BTW, you should probably remove the recipe for 1.0.20.

While at it, you can also add a TEST at the end:
```
TEST()
{
	make check
}
```

File: dev-libs/libusb/libusb-1.0.21.recipe [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	2007-2009 Daniel Drake
+	2010-2012 Peter Stuge
+	2008-2013 Nathan Hjelm
+	2009-2013 Pete Batard
+	2009-2013 Ludovic Rousseau
+	2010-2012 Michael Plante
+	2011-2013 Hans de Goede
+	2012-2013 Martin Pieuchot
+	2012-2013 Toby Gray"
+         2013-2015 Chris Dickens"
```
Comment (fbrosson) [inline dev-libs/libusb/libusb-1.0.21.recipe]: 9 white spaces -> one tab

File: dev-libs/libusb/libusb-1.0.21.recipe [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+COPYRIGHT="2001 Johannes Erdfelt
+	2007-2009 Daniel Drake
+	2010-2012 Peter Stuge
+	2008-2013 Nathan Hjelm
+	2009-2013 Pete Batard
+	2009-2013 Ludovic Rousseau
+	2010-2012 Michael Plante
+	2011-2013 Hans de Goede
+	2012-2013 Martin Pieuchot
+	2012-2013 Toby Gray"
```
Comment (fbrosson) [inline dev-libs/libusb/libusb-1.0.21.recipe]: You forgot to remove the closing quotes here.

File: dev-libs/libusb/libusb-1.0.21.recipe [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+	libusb$secondaryArchSuffix = $portVersion
+	lib:libusb_1.0$secondaryArchSuffix = 0.1.0 compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libusb${secondaryArchSuffix}_devel = $portVersion
+	devel:libusb_1.0${secondaryArchSuffix} = 0.1.0 compat >= 0
```
Comment (fbrosson) [inline dev-libs/libusb/libusb-1.0.21.recipe]: You can can drop the "{" and "}" (in the line with devel:libusb_1.0)

File: dev-libs/libusb/libusb-1.0.21.recipe [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+	2007-2009 Daniel Drake
+	2010-2012 Peter Stuge
+	2008-2013 Nathan Hjelm
+	2009-2013 Pete Batard
+	2009-2013 Ludovic Rousseau
+	2010-2012 Michael Plante
+	2011-2013 Hans de Goede
+	2012-2013 Martin Pieuchot
+	2012-2013 Toby Gray
+        2013-2015 Chris Dickens"
```
Comment (fbrosson) [inline dev-libs/libusb/libusb-1.0.21.recipe]: Please use a tab instead of white spaces.

File: dev-libs/libusb/libusb-1.0.21.recipe [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+devices on Linux, OS X, Windows and OpenBSD.
+- user-mode: No special privilege or elevation is required for the \
+application to communicate with a device.
+- version-agnostic: All versions of the USB protocol, from 1.0 to 3.0 \
+(latest), are supported."
+HOMEPAGE="http://libusb.info/"
+COPYRIGHT="2001 Johannes Erdfelt
+	2007-2009 Daniel Drake
+	2010-2012 Peter Stuge
+	2008-2013 Nathan Hjelm
```
Comment (fbrosson) [inline dev-libs/libusb/libusb-1.0.21.recipe]: Should be
```
	2008-2016 Nathan Hjelm
```

File: dev-libs/libusb/libusb-1.0.21.recipe [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+	2009-2013 Pete Batard
+	2009-2013 Ludovic Rousseau
+	2010-2012 Michael Plante
+	2011-2013 Hans de Goede
+	2012-2013 Martin Pieuchot
+	2012-2013 Toby Gray
+        2013-2015 Chris Dickens"
+LICENSE="GNU LGPL v2.1"
+REVISION="2"
```
Comment (fbrosson) [inline dev-libs/libusb/libusb-1.0.21.recipe]: You probably want to start from 1:
```
REVISION="1"
```

File: dev-libs/libusb/libusb-1.0.21.recipe (hunk 5 above)
Comment (fbrosson) [inline dev-libs/libusb/libusb-1.0.21.recipe]: Please use a tab instead of white spaces.

**Comment by Vidrep** [comment]
fbrosson, now I'm lost. Let's close this until humdinger can walk me through it tomorrow

File: dev-libs/libusb/libusb-1.0.21.recipe (hunk 7 above)
Comment (Vidrep) [inline dev-libs/libusb/libusb-1.0.21.recipe]: The first mistake I made (beside attempting this) was not working from my copy of the recipe with a proper editor. I now see the whitespaces etc you refer to.

**Comment by fbrosson** [comment]
OK.

BTW, the recipe is building fine on x86_gcc2. (I haven't tested on x86 and x86_64 yet.)

**Comment by Vidrep** [comment]
Both x86_gcc2 and x86_64 build OK. I've been using them for about a week.
I'm not a coder, so there's not much use in letting me fix the recipe.
I thought I'd try on my own, after having someone walk me through one other git push earlier.

**Comment by korli** [comment]
@Vidrep you should move your commits to a branch anyway.

File: dev-libs/libusb/libusb-1.0.21.recipe [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+	sh bootstrap.sh
+	runConfigure ./configure
+	# multiple jobs are not reliable
+	make
+}
+
+INSTALL()
+{
+	make install
+	rm $libDir/*.la
```
Comment (jessicah) [inline dev-libs/libusb/libusb-1.0.21.recipe]: Now you're just getting overly pedantic. We don't want any libtool files in the package.

**Comment by fbrosson** [comment]
Jumping to #1119.

**Pull request closed without merging**